### ISSUE TITLE
Missing pathinfo

### ISSUE THREAD

**Issue by StefanoBalocco** [monkey/monkey#240]
Hello,
         I've noticed that PATH_INFO aren't produced for php scripts.

Using this "default" sites:
```
[HOST]
        ServerName localhost
        DocumentRoot /srv/www/localhost/htdocs

[LOGGER]
        AccessLog /var/log/monkey/access.log
        ErrorLog /var/log/monkey/error.log

[ERROR_PAGES]
        404  404.html

[HANDLERS]
        Match /.*\.php.* fastcgi
        Match /.*\.php fastcgi

```
I tried to do:
```
# curl -D - localhost/index.php && echo
HTTP/1.1 200 OK
Server: Monkey/1.6.8
Date: Fri, 13 May 2016 09:42:26 GMT
Transfer-Encoding: Chunked
Content-type: text/html; charset=UTF-8

array(23) {
  ["USER"]=>
  string(8) "www-data"
  ["HOME"]=>
  string(8) "/var/www"
  ["FCGI_ROLE"]=>
  string(9) "RESPONDER"
  ["GATEWAY_INTERFACE"]=>
  string(7) "CGI/1.1"
  ["REDIRECT_STATUS"]=>
  string(3) "200"
  ["SERVER_SOFTWARE"]=>
  string(12) "Monkey/1.6.8"
  ["SERVER_PROTOCOL"]=>
  string(8) "HTTP/1.1"
  ["SERVER_NAME"]=>
  string(9) "localhost"
  ["DOCUMENT_ROOT"]=>
  string(25) "/srv/www/localhost/htdocs"
  ["SERVER_ADDR"]=>
  string(9) "127.0.0.1"
  ["SERVER_PORT"]=>
  string(2) "80"
  ["REMOTE_ADDR"]=>
  string(9) "127.0.0.1"
  ["REMOTE_PORT"]=>
  string(5) "59889"
  ["SCRIPT_FILENAME"]=>
  string(35) "/srv/www/localhost/htdocs/index.php"
  ["SCRIPT_NAME"]=>
  string(10) "/index.php"
  ["REQUEST_METHOD"]=>
  string(3) "GET"
  ["REQUEST_URI"]=>
  string(10) "/index.php"
  ["HTTP_USER_AGENT"]=>
  string(11) "curl/7.38.0"
  ["HTTP_HOST"]=>
  string(9) "localhost"
  ["HTTP_ACCEPT"]=>
  string(3) "*/*"
  ["PHP_SELF"]=>
  string(10) "/index.php"
  ["REQUEST_TIME_FLOAT"]=>
  float(1463132546.5885)
  ["REQUEST_TIME"]=>
  int(1463132546)
}

# curl -D - localhost/index.php/a && echo
HTTP/1.1 404 Not Found
Server: Monkey/1.6.8
Date: Fri, 13 May 2016 09:42:29 GMT
Content-Type: text/html
Content-Length: 210

<HTML><HEAD><STYLE type="text/css"> body {font-size: 12px;} </STYLE></HEAD><BODY><H1>Not Found</H1>The requested URL was not found on this server.<BR><HR><ADDRESS>Powered by Monkey/1.6.8</ADDRESS></BODY></HTML>
#
```

The first one doesn't have the empty PATH_INFO, the second one doesn't work at all.

**Comment by edsiper** [comment]
Hi Stefano,

I have reviewed your report and the problem it self it's not PATH_INFO.

Monkey in 1.6 used to "validate" an existent static file before to pass the control to some plugin handler, this was fixed in the development version of Monkey 1.7. To make things easier I have backported this solution to our 1.6 branch.

The solution will be delivered in our "delayed" v1.6.9, for now you can grab a copy from here:

  https://github.com/monkey/monkey/tree/1.6

> Note: PATH_INFO is a hack and evil, PHP provides a cgi.fix_pathinfo configuration flag to make it work properly without risks.

**Comment by edsiper** [comment]
Closing this issue, if it persists please re-open it.
